src/main/java/bio/terra/service/resourcemanagement/AzureDataLocationSelector.java

```diff
@@ -0,0 +1,52 @@
+package bio.terra.service.resourcemanagement;
+
+import bio.terra.common.exception.NotImplementedException;
+import bio.terra.model.BillingProfileModel;
+import org.springframework.context.annotation.Primary;
+import org.springframework.stereotype.Component;
+
+import java.nio.charset.StandardCharsets;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+
+@Primary
+@Component
+public class AzureDataLocationSelector {
```

Comment:
Why is this an injectable? Both methods can be static.

Reply:
To make it match with the GCP based locator